data/json/items/melee/knives_kitchen.json

```
@@ -65,6 +117,25 @@
     "flags": [ "SHEATH_KNIFE", "ALLOWS_BODY_BLOCK" ],
     "melee_damage": { "bash": 1, "stab": 6 }
   },
+  {
+    "id": "knife_paring_ti",
+    "name": { "str": "titanium paring knife", "str_pl": "titanium paring knives" },
+    "type": "TOOL",
+    "category": "weapons",
+    "description": "A high-performance kitchen knife with a short blade, made for peeling vegetables with precise cuts.  Features a wear-resistant titanium alloy blade, durable construction, and even an antimicrobial handle.",
+    "ascii_picture": "knife_paring",
+    "copy-from": "base_titanium_kitchen_knife",
+    "looks_like": "knife_paring",
+    "weight": "28 g",
+    "volume": "30 ml",
+    "longest_side": "15 cm",
+    "price": 500,
+    "price_postapoc": 100,
+    "to_hit": { "grip": "weapon", "length": "hand", "surface": "point", "balance": "neutral" },
+    "qualities": [ [ "CUT", 2 ], [ "CUT_FINE", 2 ], [ "BUTCHER", 12 ] ],
```

```suggestion
    "qualities": [ [ "CUT", 2 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 12 ] ],
```
Why would the blade material change its fine cutting :thinking:

---

Paring knives in general are designed for controlled small cuts, in a way standard knives aren't (they're much more like an x-acto knife in that way). I'd like to update steel paring knives with fine cutting 2 as well, but I was going to handle that separately. Thoughts?

---

Just went ahead and removed Fine Cutting 2 for now, and will consider adding it to all paring knives in separate PR.